support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -286,6 +287,21 @@ function OneTimeCheckoutComponent({
 					setErrorMessage('Sorry, something went wrong.');
 					setErrorContext(appropriateErrorMessage(paymentResult.error ?? ''));
 				}
+				if (paymentResult.paymentStatus === 'success') {
+					const order = {
+						firstName: '',
+						paymentMethod: paymentMethod,
+					};
+					setThankYouOrder(order);
+
+					/*  Contribution product reused for both
+              OneOff (excl ratePlan)
+              Recurring (incl ratePlan) */
+					const thankYouUrlSearchParams = new URLSearchParams();
+					thankYouUrlSearchParams.set('product', 'Contribution');
```

Comment:
This is reserved for when the product is actually `Contribution` as per the [Product API](https://product-catalog.guardianapis.com/product-catalog.json). 

We shouldn't try to hijack this.

Reply:
Agree, thats why a description was placed there to highlight the assumption. Hijack is not ideal. I do not think product=contributiom with no RatePlan combination is used in the code-base hence it was selected for ease of fitting in rather reflecting the setup. Meeting is need to progress this further would say however.